src/test/java/com/wikia/webdriver/common/core/Helios.java

```diff
@@ -192,7 +182,7 @@ private static String getTokenFromCache(String userName) {
   }
 
   private static String getUserId(String userName) {
-    String getUserIDURL = "";
+    String getUserIDURL;
     try {
       getUserIDURL = String.format("%s/api.php?action=query&list=users&ususers=%s&format=json&cb=%d",
```

Comment:
You can extract this variable evaluation into another method, so you don't have a hanging variable declaration. The variable could also be final.